src/main/webapp/app/quiz/edit/short-answer-question/edit-short-answer-question.component.ts

```diff
@@ -81,14 +87,20 @@ export class EditShortAnswerQuestionComponent implements OnInit, OnChanges, Afte
         if (changes.question && changes.question.previousValue != null) {
             this.questionUpdated.emit();
         }
+        /** Update backupQuestion if the question changed **/
```

Comment:
I don't think this is correct. Otherwise each change, also changes the backup question and you can never return to the original question. Or what is backupQuestion used for?

Reply:
Yeah, you are right. I tested it and it actually does not get updated at all (I added it as it was also in the reevaluation of DnD :see_no_evil: ).
So setting the backupQuestion in the ngOnInit is enough. I will remove those lines of code.